python/lsst/sconsUtils/tests.py

```diff
@@ -293,3 +305,38 @@ def junitPrefix(self):
             prefix += ".{0}".format(os.environ[controlVar])
 
         return prefix
+
+    def _pytestCoverage(self, prefix="pytest"):
+        """Form the additional arguments required to enable coverage testing.
+        Prefix is used to form the output files.
+        """
+
+        options = ""
+
+        # if we have a standard package layout, restrict coverage to the
+        # python code and do not include the tests.
+        if os.path.exists("python") or os.path.exists(os.path.join(os.path.pardir, "python")):
```

Comment:
Why these two? Could you instead just use `eups.packagedir()` here to take care of both cases?

Reply:
I've changed it to always use `.` and include coverage of tests.